omod/src/main/java/org/openmrs/module/fhir2/narrative/OpenMRSNarrativeTemplateManifest.java

```diff
@@ -0,0 +1,243 @@
+			
+			if (nextKey.endsWith(".profile")) {
+				String profile = file.getProperty(nextKey);
+				if (isNotBlank(profile)) {
+					nextTemplate.addAppliesToProfile(profile);
+				}
+			} else if (nextKey.endsWith(".resourceType")) {
+				String resourceType = file.getProperty(nextKey);
+				Arrays.stream(resourceType.split(",")).map(t -> t.trim()).filter(t -> isNotBlank(t))
+				        .forEach(t -> nextTemplate.addAppliesToResourceType(t));
+			} else if (nextKey.endsWith(".dataType")) {
+				String dataType = file.getProperty(nextKey);
+				Arrays.stream(dataType.split(",")).map(t -> t.trim()).filter(t -> isNotBlank(t))
+				        .forEach(t -> nextTemplate.addAppliesToDatatype(t));
+			} else if (nextKey.endsWith(".style")) {
+				String templateTypeName = file.getProperty(nextKey).toUpperCase();
+				TemplateTypeEnum templateType = TemplateTypeEnum.valueOf(templateTypeName);
+				nextTemplate.setTemplateType(templateType);
+			} else if (nextKey.endsWith(".contextPath")) {
+				String contextPath = file.getProperty(nextKey);
+				nextTemplate.setContextPath(contextPath);
+			} else if (nextKey.endsWith(".narrative")) {
+				String narrativePropName = name + ".narrative";
+				String narrativeName = file.getProperty(narrativePropName);
+				if (isNotBlank(narrativeName)) {
+					nextTemplate.setTemplateFileName(narrativeName);
+				}
+			} else if (nextKey.endsWith(".title")) {
+				ourLog.debug("Ignoring title property as narrative generator no longer generates titles: {}", nextKey);
+			} else {
+				throw new ConfigurationException("Invalid property name: " + nextKey
+				        + " - the key must end in one of the expected extensions "
+				        + "'.profile', '.resourceType', '.dataType', '.style', '.contextPath', '.narrative', '.title'");
+			}
+			
+		}
+		
+		return nameToTemplate.values();
+	}
+	
+	static String loadResource(String name) throws IOException {
+		if (name.startsWith("classpath:")) {
+			String cpName = name.substring("classpath:".length());
+			try (InputStream resource = DefaultThymeleafNarrativeGenerator.class.getResourceAsStream(cpName)) {
+				if (resource == null) {
+					try (InputStream resource2 = DefaultThymeleafNarrativeGenerator.class
+					        .getResourceAsStream("/" + cpName)) {
+						if (resource2 == null) {
+							throw new IOException("Can not find '" + cpName + "' on classpath");
+						}
+						return IOUtils.toString(resource2, Charsets.UTF_8);
+					}
+				}
+				return IOUtils.toString(resource, Charsets.UTF_8);
+			}
+		} else if (name.startsWith("file:")) {
+			File file = new File(name.substring("file:".length()));
+			if (file.exists() == false) {
+				throw new IOException("File not found: " + file.getAbsolutePath());
+			}
+			try (FileInputStream inputStream = new FileInputStream(file)) {
+				return IOUtils.toString(inputStream, Charsets.UTF_8);
+			}
+		} else if (name.startsWith("openmrs:")) {
```

Comment:
Here is the main modification which has been done to support `openmrs:` prefixed relative paths.